src/template/.gitignore.template

```diff
@@ -0,0 +1,13 @@
+*.js
+!jest.config.js
+!eslintrc.js
+*.d.ts
+node_modules
+dist
+
+# CDK asset staging directory
+.cdk.staging
+cdk.out
+
+# Parcel default cache directory
+.parcel-cache
```

Comment:
Do we use [parcel](https://parceljs.org/)?

Reply:
Doesn't look like it 👍